src/esp32/esp32_wifi.c

```diff
@@ -327,6 +327,31 @@ bool mgos_wifi_dev_sta_setup(const struct mgos_config_wifi_sta *cfg) {
     tcpip_adapter_dhcpc_start(TCPIP_ADAPTER_IF_STA);
   }
 
+  LOG(LL_INFO, ("Setting STA protocol: %s", cfg->protocol));
+  if (strcmp(cfg->protocol, "B") == 0) {
```

Comment:
move this to a common function.
keep style consistent (format manually or run clang-format)

Reply:
Moved to mgos_wifi_protocol_setup